src/bin/make_data.js

```diff
@@ -9,11 +9,15 @@ import path from 'path';
 
 import * as _ from 'lamb';
 import yaml from 'js-yaml';
-import {readDir, readFile, saveObj} from '@svizzle/file';
+import {readDir, readFile, saveObj, saveString} from '@svizzle/file';
 import {tapMessage} from '@svizzle/dev';
 import {applyFnMap} from '@svizzle/utils';
 
-import {indexById, makeDatasetBySource} from 'app/utils';
+import {indexById, makeDatasetBySource, generateTypingsForAll} from 'app/utils';
+
+const ESTYPINGS_PATH = path.resolve(__dirname, '../node_modules/app/elasticsearch/typings/es.ts');
```

Comment:
`app/elasticsearch/typings/es.ts` is missing so running `make_data.js` fails and in the app we get in console:
`GET http://localhost:3000/dsl/datasets.ts 404 (Not Found)`

Reply:
That was an omission, I'll push the file now